packages/@webex/plugin-meetings/test/unit/spec/reachability/request.js

```diff
@@ -115,4 +135,4 @@ describe('plugin-meetings/reachability', () => {
       assert.notCalled(webex.internal.newMetrics.callDiagnosticLatencies.measureLatency);
     });
   });
-});
+});
```

Comment:
having one test that checks that it works fine with webex.config.support.appType and webex.config.support.appVersion being undefined would be nice